core/activity/stdout_repository.go

```diff
@@ -21,3 +21,8 @@ func NewStdoutRepository(writer io.Writer) *StdoutRepository {
 func (r StdoutRepository) Insert(ctx context.Context, log *audit.Log) error {
 	return json.NewEncoder(r.writer).Encode(log)
 }
+
+// TODO : Fix this
+func (r StdoutRepository) List(ctx context.Context, filter Filter) ([]audit.Log, error) {
+	return []audit.Log{}, nil
+}
```

Comment:
need to implement List(), what kind of behaviour should be here?

Reply:
We can return err not found since we won't support this List action if sink is stdout log